content/docs/addons-test-utils.md

```diff
@@ -1,27 +1,27 @@
 ---
 id: test-utils
-title: Test Utilities
+title: परीक्षण उपयोगिताएँ
 permalink: docs/test-utils.html
 layout: docs
 category: Reference
 ---
 
-**Importing**
+**इम्पॉर्टिंग**
 
 ```javascript
 import ReactTestUtils from 'react-dom/test-utils'; // ES6
-var ReactTestUtils = require('react-dom/test-utils'); // ES5 with npm
+var ReactTestUtils = require('react-dom/test-utils'); // ES5, npm के द्वारा
 ```
 
-## Overview {#overview}
+## ओवरव्यू {#overview}
 
-`ReactTestUtils` makes it easy to test React components in the testing framework of your choice. At Facebook we use [Jest](https://facebook.github.io/jest/) for painless JavaScript testing. Learn how to get started with Jest through the Jest website's [React Tutorial](https://jestjs.io/docs/tutorial-react).
+`ReactTestUtils` आपकी पसंद के परीक्षण फ्रेमवर्क में React कौम्पोनॅन्ट का परीक्षण करना आसान बनाता है। Facebook में हम पीड़ारहित जावास्क्रिप्ट परीक्षण के लिए [Jest](https://facebook.github.io/jest/) का उपयोग करते हैं। Jest वेबसाइट के [React ट्यूटोरियल](https://jestjs.io/docs/tutorial-react) के माध्यम से Jest के साथ शुरुआत करने का तरीका जानें।
 
-> Note:
+> नोट:
 >
-> We recommend using [React Testing Library](https://testing-library.com/react) which is designed to enable and encourage writing tests that use your components as the end users do.
+> हम [React Testing Library](https://testing-library.com/react) का उपयोग करने की सलाह देते हैं, जो अंतिम उपयोगकर्ताओं के रूप में आपके कौम्पोनॅन्ट का उपयोग करने वाले लेखन परीक्षणों को सक्षम और प्रोत्साहित करने के लिए डिज़ाइन किया गया है।
```

Comment:
this is a little complex translation, could we simplify it a bit?

Reply:
I agree, but unable to frame the sentence - it is so complex

Reply:
हम [React Testing Library](https://testing-library.com/react) का उपयोग करने की सलाह देते हैं। Yeh library is prakar design ki gai hai ki aap tests ussi prakar likhe jis prakar अंत उपयोगकर्ता usse istemaal karenge